tools/model_tools/src/openvino/model_zoo/download_engine/downloader.py

```diff
@@ -213,6 +213,12 @@ def _download_model(self, reporter, session_factory, model, known_precisions: se
 
         reporter.print_group_heading('Downloading {}', model.name)
 
+        model_unsupported_precisions = self.requested_precisions - model.precisions
+        if model_unsupported_precisions:
+            reporter.print_section_heading('Model {} is not supported in {} precisions. '
+                + 'Skipping downloading for these precisions.', model.name, model_unsupported_precisions)
+            reporter.print()
```

Comment:
I think the message should be an alternative to `reporter.print_group_heading('Downloading {}', model.name)`, not a follow up.